src/svgeditor/ImageEdit.as

```diff
@@ -824,18 +824,7 @@ package svgeditor {
 			}
 		}
 
-		private function clearSelection():void {
-			if (this is BitmapEdit) {
```

Comment:
Instead of sticking with this type check, I just made the method `protected` and moved the contents into `BitmapEdit`.